mos_tests/neutron/python_tests/test_dvr.py

```diff
@@ -269,7 +370,101 @@ def test_routing_after_reset_computes(self, prepare_openstack, env_name):
         self.reset_computes(self.compute_nodes, env_name)
 
         time.sleep(60)
+
         # Check ping after reset
         self.check_ping_from_vm(vm=self.server2,
                                 vm_keypair=self.instance_keypair,
                                 ip_to_ping=self.server1_ip)
+
+    @pytest.mark.check_env_('is_ha')
+    def test_east_west_connectivity_after_destroy_controller(
+            self, prepare_openstack, env_name):
+        """Check East-West connectivity after destroy controller
+
+        Scenario:
+            1. Create net01, subnet net01__subnet for it
+            2. Create net02, subnet net02__subnet for it
+            3. Create router01_02 with router type Distributed and
+                with gateway to external network
+            4. Add interfaces to the router01_02 with net01_subnet and
+                net02_subnet
+            5. Boot vm_1 in the net01
+            6. Boot vm_2 in the net02
+            7. Check that VMs are on the different computes (otherwise migrate
```

Comment:
There is no 7th step in code logic. Instead you just boot VMs on different computes. Let's change 6th step description (like at other compute) and remove 7th step